pkg/controllers/dynakube/activegate/conditions/conditions.go

```diff
@@ -0,0 +1,70 @@
+package conditions
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	ReasonCreated         string = "Created"
+	ReasonError           string = "Error"
+	ReasonUnexpectedError string = "UnexpectedError"
+	ReasonUpToDate        string = "UpToDate"
+)
```

Comment:
Try to use actual error conditions, stuff like whats in the common conditions package: https://github.com/Dynatrace/dynatrace-operator/blob/37afc6130005c92ba0ff4fddb503292be48269aa/pkg/util/conditions/error.go#L13-L32